retrospective-update/functions.py

```
@@ -0,0 +1,1065 @@
+
+            # Use s5cmd to sync the output directory to S3
+            s3_path = f'{S3_HYDROSOS_DIR}/year={year}/month={str(month).zfill(2)}/'
+            result = subprocess.run(
+                f's5cmd '
+                f'--credentials-file {credentials} --profile odp '
+                f'sync '
+                f'{output_dir} '
+                f'{s3_path}',
+                shell=True, capture_output=True, text=True,
+            )
+            if not result.returncode == 0:
```

Ruff (0.12.2)

Use `result.returncode != 0` instead of `not result.returncode == 0`

Replace with `!=` operator

(SIM201)